### DISCUSSION TITLE
Is anything being done about text needing to load in carousel on quick scroll?

### DISCUSSION THREAD

**Discussion by NafeeJ** [ppy/osu#13715] [Q&A]
On stable, when you quick scroll with right click, there is zero loading time for text in the carousel but on lazer you have to wait a second for the text to load. (Thumbnails/Images also take time to load but I feel thats a big enough I/O operation to be forgiven)

Is this known or should I move this to the issues tab?

Screenshot while quick scrolling on lazer:
![osu_2021-06-30_19-04-18](https://user-images.githubusercontent.com/33550260/124042103-2b713400-d9d6-11eb-9b81-c7955137fba0.jpg)

Screenshot while quick scrolling on stable:
![screenshot1692](https://user-images.githubusercontent.com/33550260/124042586-2660b480-d9d7-11eb-84bb-f2c017da6d02.png)

**Comment by peppy** [comment]
Text is also an IO operation, especially for unicode strings.

This may improve in the future, but is intentional for now. It was the cause of stutters and excess allocations on stable.

**Comment by LittleEndu** [comment]
Maybe some sort of a placeholder could be designed for beatmap panels. To me, the weird part is seeing fully transparent boxes scrolling by and not the part where they have no text.

**Reply by Moltanicaa** [reply]
Or maybe just draw only the backgrounds?

**Reply by peppy** [reply]
The backgrounds *are* the "placeholder". The design of the screen is going to change so this is not the time to change or dwell on this.
